NSS-Evening-E22/gitsub-hex-clan-e22
# navbar and footer as components.  Images folder with github logo

## Description
Created NavBar and Footer based on design specs.  HTML was moved into .js files of same name (respectively) and main.js updated to render contents on the DOM.

## Related Issue
#16 
#17

## Motivation and Context
Navigation and footer are required.

## How Can This Be Tested?
Checkout branch sb_ISSUE#17_footer.  Open index.html in browser.  Nav and Footer components should render on screen.

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/121408336/217978194-47fe0ee5-d4dc-4562-abd9-93d4740936ff.png)

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
